src/main/app/drafts/models/draftLegalClaim.ts

```diff
@@ -30,7 +30,7 @@ export default class DraftLegalClaim implements Serializable<DraftLegalClaim> {
 
   deserialize (input: any): DraftLegalClaim {
     if (input) {
-      this.externalId = input.externalId
+      this.externalId = input['externalId']
```

Comment:
this is wrong, it should be on a class

Reply:
Done.

Reply:
it's not wrong, we have removed DraftDocument class

Reply:
And external Id is draft JavaScript object

Reply:
still wrong and shouldn't be doing it